web_src/js/features/comp/SearchUserBox.js

```diff
@@ -17,14 +17,13 @@ export function initCompSearchUserBox() {
         const searchQuery = $searchUserBox.find('input').val();
         const searchQueryUppercase = searchQuery.toUpperCase();
         $.each(response.data, (_i, item) => {
-          let title = item.login;
-          if (item.full_name && item.full_name.length > 0) {
-            title += ` (${htmlEscape(item.full_name)})`;
-          }
           const resultItem = {
-            title,
+            title: item.login,
             image: item.avatar_url
           };
+          if (item.full_name && item.full_name.length > 0) {
+            resultItem.description = htmlEscape(item.full_name);
+          }
```

Comment:
```suggestion
          if (item.full_name) {
            resultItem.description = htmlEscape(item.full_name);
          }
```

`if (item.full_name)` already implies `item.full_name.length > 0`

Reply:
Probably better to not try with `null`.